VISUALIZERS/PLOTLY/PROPHET_COMPONENTS/PROPHET_COMPONENTS.py

```diff
@@ -1,7 +1,7 @@
 from flojoy import flojoy, run_in_venv, DataFrame, Plotly, DataContainer
 
 
-@flojoy(deps={"prophet": "1.1.4", "holidays": "0.26", "pystan": "2.19.1.1"})
```

Comment:
Why remove these?

Reply:
We have `run_in_venv` decorator which will execute this node in a separate venv so using deps parameter of flojoy is redundant here.